<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryViewClass</class>
 <widget class="QWidget" name="QueryViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>16777215</height>
   </size>
  </property>
  <widget class="QLabel" name="query_view_drop_down">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>20</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="query_view_back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>22</y>
     <width>756</width>
     <height>138</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>756</width>
      <height>138</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../UmInstallerAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
